doc/quick-start.md

```diff
@@ -123,3 +123,53 @@ df = result.DataFrame()
 ```{code-cell} ipython3
 df.head()
 ```
+
+# Jupyter Magic
+
+One may be unfamiliar with the commands prefixed with '%' used in this instruction. Here is a detailed description of this command and its usage. 
+
+## Definition of Jupyter Magic
+
+Magics are specific to and provided by the IPython kernel. Some common usage of magic functions are: running external code files, timing code execution, and loading IPython Extensions. 
+
+```{code-cell} ipython3
+
+# Suppose execute.py is a python code file
+
+%run execute.py
```

Comment:
This is the main reason why it builds fail, since every code inside `{code-cell} ipython3` section will actual run and look up for `execute.py` file to execute

If you just want to explain the usage then you can just move it out this code-cell section & use regular markdown format to explain

Reply:
@chuwpeng123 please address this, you need to convert this to markdown cells - this will prevent our doc building process for running them. Also, I'd suggest moving this to a new file called FAQ and put it at the end of the user guide, then link from the quickstart to the FAQ document via an [admonition](https://jupyterbook.org/en/stable/reference/cheatsheet.html#admonitions)

more info here: https://github.com/ploomber/contributing/blob/main/documentation/notebooks.md

Reply:
@edublancas I am looking at it. Try to get this down by tomorrow.